frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -504,6 +505,9 @@ const plugin: Plugin<ConsumedExtensions> = [
           )
         ).default,
     },
+    flags: {
+      disallowed: [FEATURES.SS_LIST],
+    },
```

Comment:
don't we want to disallow add-capacity extension as well in this plugin file ??

Reply:
yeah just saw it, don't know why we have it in both the places though.

Reply:
IIRC, this one is for add-capacity kebab on `Installed Operators> Operator details` page.